.github/workflows/backport.yaml

```
@@ -0,0 +1,45 @@
+name: Backport
+
+on:
+  pull_request_target:
+    types: [closed, labeled]
+
+permissions:
+  contents: read
+
+jobs:
+  pull-request:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    if: github.repository_owner == 'akuity' && github.event.pull_request.merged && (github.event_name != 'labeled' || startsWith('backport/', github.event.label.name))
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Create backport PRs
+        uses: korthout/backport-action@v2
+        # xref: https://github.com/korthout/backport-action#inputs
+        with:
+          # Use token to allow workflows to be triggered for the created PR
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          # Match labels with pattern `backport/<target-branch>`
+          label_pattern: '^backport\/([^ ]+)$'
```

Shouldn't it be
```suggestion
          label_pattern: '^backport\/(.+)$'
```
? 👀

---

That would also match spaces, which are not allowed as branch names.